lib/nylas/delta.rb

```diff
@@ -42,8 +45,10 @@ def cast(data)
              else
                data
              end
+
+      data = data.merge(data[:attributes]) if data[:attributes]
```

Comment:
Random question: What would happen if you merged a nil or missing hash?

Reply:
If it's `nil` it will raise an error and that's why we have `if` condition. But if it's empty it will do nothing. I will try to add test for these condition as well so it will not break.